vagrant-scripts/vitess/build.sh

```diff
@@ -14,6 +14,10 @@ printf "\nBuilding Vitess...\n"
 sudo chown "$(whoami)":"$(whoami)" /vagrant
 sudo chown "$(whoami)":"$(whoami)" /vagrant/src
 cd "$VITESS_WORKSPACE"
+
+# open-jdk version that we are using in the VM needs this flag, otherwise we will fail to build ZK
+export JAVA_TOOL_OPTIONS="-Dhttps.protocols=TLSv1.2"
```

Comment:
in vagrant we install `openjdk-7-jdk` while in Docker [we use](https://github.com/vitessio/vitess/blob/master/docker/bootstrap/Dockerfile.common#L16) `openjdk-8-jdk`.

This is another example for https://github.com/vitessio/vitess/issues/4780

Reply:
Yeah. This is a limitation of the ubuntu version we are using in the Vagrant setup. We need to do some upgrades. openjdk-8 is not available. I think we should do that as part of a different PR.